test_metannotate_end_to_end.sh

```diff
@@ -32,13 +32,13 @@ for entry in $(ls test_output); do
     # Checks if entry is the one we are looking to compare
     if [[ ${entry} == *"0_msa"* && ${entry} == *".fa"* ]]; then
         cd test_output
-        GENERATED_FILES[0]=$(md5sum ${entry} | awk '{ print $1 }')
+        GENERATED_FILES[0]=$(md5sum ${entry} | cut -d ' ' -f 1)
         cd ..
     fi
     # Checks if entry is the one we are looking to compare
     if [[ ${entry} == *"refseq"* && ${entry} == *".fa"* ]]; then
         cd test_output
-        GENERATED_FILES[1]=$(md5sum ${entry} | awk '{ print $1 }')
+        GENERATED_FILES[1]=$(md5sum ${entry} | cut -d ' ' -f 1)
```

Comment:
@KyleBehiels Rename this variable to regenerated file *hashes* or something like that